src/common/utilities.ts

```
@@ -2,3 +2,15 @@
 export function degreesToRadians(degrees: number): number {
   return (degrees / 360) * (2 * Math.PI);
 }
+
+export function hexToColor(hex: string): [number, number, number] {
```

I noticed that this function fails silently, should we change this?

---

I don't intend this PR to change any behaviour so let's not; we can keep it as an issue